recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "3.8.1" %}
+{% set version = "4.0.0" %}
 
 package:
   name: arcsi
   version: {{ version }}
 
 source:
-  url: https://github.com/remotesensinginfo/arcsi/archive/v{{ version }}.tar.gz
-  sha256: 83b069487e363b26de6a2708256b55846e19a6a8a906086ac025152268208877
+  url: https://github.com/remotesensinginfo/arcsi/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ff78ccbb4622c4ec59b66f0309d5716ba2c70b0b13b8df53e75909f27380ee2b
 
 build:
-  number: 2
-  skip: True  # [win and py27]
+  number: 0
+  skip: True  # [win]
```

Comment:
Was this change intentional?
Previously this only skipped Python 2.7 on Windows (since we don't build for Python 2.7 for quite a while, this was essentially ignored).
Now no Windows builds are created at all. To re-enable Windows builds you could issue a PR with simply:
```suggestion
```